### PR TITLE
Added basic dummy players

### PR DESCRIPTION
**Pull request by Megaleo**
When there are fewer than 3 players when the master calls /startgame,
they are presented with the option to include dummies. They choose
everything randomly.

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -105,6 +106,14 @@ def menu_card(card, player, text=None, clue=None):
...
             )
 
+def random_card_msg(player, card_list):
+    '''Returns message with a random chosen card from card_list'''
+    card_id = choice(card_list).id
+    user_id = player.id
+    return f"{user_id}:{card_id}"
+
+def random_card_from_hand(player):
+    return random_card_msg(player, player.hand)
```
[the remark this answers is not in the document]
Comment (gb-solis) [reply utils.py:116]: Nevermind, reading the changes on `main` I understood it. Good thinking!

File: main.py [hunk 2]
```diff
@@ -127,19 +189,35 @@ def query_callback(update, context):
...
 
+        query.answer(text='Settings saved!')
         if data in [c.name for c in dixit_game.end_criteria]:
             dixit_game.end_criterion = dixit_game.end_criteria[data]
 
         query.edit_message_text(text=text, reply_markup=markup)
 
-    if query.data.isdecimal(): # if the user is sending us endgame values
-        dixit_game.end_criterion_number = int(query.data)
+    if query.data.startswith('end number'):
```
Comment (gb-solis) [inline main.py:198]: Now that every type of query data is of the form `type:value` we could split it in `type` and `values` early, to avoid repeating code

**Review by gb-solis — APPROVED** [review]
:+1:

File: main.py (hunk 2 above)
Comment (Megaleo) [reply main.py:198]: That's a good ideia!

**Pull request merged**